igor.dzetcyna/first/task 1/array.rb

```diff
@@ -0,0 +1,8 @@
+# Заповнити випадковими числами від -10 до 10 двовимірний масив 3х3 елементів
+arr = Array.new(3) { Array.new(3) { rand -10..10 } }
+p arr
+# Перетворимо багатовимірний масив в одновимірний і знаходимо все позитивні числа
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#english-comments
Please write all comments in english